### PR TITLE
Fix cache delete bug master

### PR DESCRIPTION
**Pull request by shimonewman**
fix delete cache bug to master

### CODE REVIEW

**Review by stnguyen90 — APPROVED** [review]

File: app/controllers/api/storage.php [hunk 1]
```diff
@@ -654,11 +653,6 @@
             ->setContext('bucket', $bucket)
         ;
 
-        $deletes
```
Comment (eldadfux) [inline app/controllers/api/storage.php:657]: Didn't we have this to avoid a conflict when a file with a similar ID was recently deleted?

Comment (stnguyen90) [reply app/controllers/api/storage.php:657]: @eldadfux this event should already be triggered when the file is deleted, right?

File: app/workers/deletes.php [hunk 2]
```diff
@@ -130,31 +130,56 @@ public function shutdown(): void
...
-        $this->deleteCacheFiles([
-            Query::lessThan('accessedAt', $this->args['datetime']),
-        ]);
+        $cache = new Cache(
+            new Filesystem(APP_STORAGE_CACHE . DIRECTORY_SEPARATOR . 'app-' . $projectId)
+        );
+
+        $query = [
+            Query::equal('resource', [$resource])
+        ];
```
Comment (stnguyen90) [inline app/workers/deletes.php:148]: Wait...how does this handle the same file id in multiple buckets?

**Review by stnguyen90 — COMMENTED** [review]
See https://github.com/appwrite/appwrite/pull/5698/files#r1269913474

**Review by stnguyen90 — APPROVED** [review]
Approving this, but created https://github.com/appwrite/appwrite/issues/5861

**Review by eldadfux — APPROVED** [review]

**Pull request merged**